src/main/java/de/tum/in/www1/artemis/web/rest/dto/DomainObjectIdDTO.java

```diff
@@ -0,0 +1,13 @@
+package de.tum.in.www1.artemis.web.rest.dto;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import de.tum.in.www1.artemis.domain.DomainObject;
+
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+public record DomainObjectIdDTO(Long id) {
```

Comment:
For discussion as well: I feel like the TeamScale comment and the Codacy comment both are invalid here. Can we generally suppress them?

Reply:
I also added the former  as a discussion point in this PR https://github.com/ls1intum/Artemis/pull/7027 (@MaximilianAnzinger)